nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/test/java/org/apache/nifi/services/azure/storage/TestADLSCredentialsControllerService.java

```diff
@@ -327,6 +327,28 @@ public void testGetCredentialsDetailsWithSasToken() throws Exception {
         assertNull(actual.getServicePrincipalClientSecret());
     }
 
+    @Test
+    public void testGetCredentialsDetailsWithSasTokenUsingEL() throws Exception {
+        // GIVEN
+        configureAccountName();
+        configureSasTokenUsingEL();
+
+        runner.enableControllerService(credentialsService);
+
+        // WHEN
+        ADLSCredentialsDetails actual = credentialsService.getCredentialsDetails(new HashMap<>());
+
+        // THEN
```

Comment:
Although a few tests include the given/when/then comments, I recommend avoiding them for new tests.  If there are no other changes, then it is fine to leave them in place, but otherwise I think it would be better to avoid the comments.

Reply:
Hi 
Can we merge this change to proceed further?  I need this feature to test our on one of the solution built on Nifi.